frontend/src/components/organisms/TableFilters/index.jsx

```diff
@@ -72,28 +109,37 @@ const TableFilter = (props) => {
   return (
     <>
       <Toolbar id="filter-toolbar">
-        {filterData?.length > 0 && (
-          <ToolbarContent className="field-filter">
-            <ToolbarItem style={{ marginInlineEnd: 0 }}>
-              <SelectBox
-                options={filterData}
-                onChange={onCategoryChange}
-                selected={categoryFilterValue}
-                icon={<FilterIcon />}
-                width={"200px"}
-              />
-            </ToolbarItem>
-            <ToolbarItem>
-              <MultiSelectBox
-                options={filterOptions}
-                onChange={updateSelectedFilter}
-                applyMethod={onOptionsChange}
-                currCategory={category}
-                selected={selectedFilters?.find((i) => i.name === category)}
-                width={"300px"}
-              />
-            </ToolbarItem>
-          </ToolbarContent>
+        {tableFilters != null && filterOptions != null && updateSelectedFilter != null && (
+          tableFilters.length > 0 && filterOptions.length > 0 ? (
+            <ToolbarContent className="field-filter">
+              <ToolbarItem style={{ marginInlineEnd: 0 }}>
+                <SelectBox
+                  options={tableFilters}
+                  onChange={onCategoryChange}
+                  selected={categoryFilterValue}
+                  icon={<FilterIcon />}
+                  width={"200px"}
+                />
+              </ToolbarItem>
+              <ToolbarItem>
+                {category != false ? (
```

Comment:
`{category ? ( ... )}`?? But while `""` might be falsey in Javascript, `"no-filter"` definitely isn't. If this is what you want, a comment would help...